docs/examples/cookbook/light-and-dark-theme.md

```diff
@@ -28,7 +28,7 @@ local scope = scoped(Fusion)
 Theme.current = scope:Value("light")
 Theme.dynamic = {}
 for colour, variants in Theme.colours do
-	Theme.dynamic[colour] = scope:Computed(function(use)
+	Theme.dynamic[colour] = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
	Theme.dynamic[colour] = scope:Computed(function(use)
```

Reply:
reverted